src/components/ResetPassword.js

```diff
@@ -0,0 +1,54 @@
+// ForgotPassword.js
+import React, { useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import { resetPassword } from '../apis/authApi'; // Make sure to import your API call function
+
+const ResetPassword = () => {
+  const history = useHistory();
```

Comment:
`useHistory` was deprecated after React Router v5 (we are using v6 if you check our package.json): https://github.com/remix-run/react-router/blob/main/docs/upgrading/v5.md#use-usenavigate-instead-of-usehistory

Reply:
This error when trying to navigate to this component is saying that `useHistory` is not a valid variable/function exported from the `react-router-dom` library:

<img width="1790" alt="Screenshot 2023-11-13 at 10 01 21 AM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/fee4dec0-007c-444e-8d03-016a50c559c5">

Reply:
Thank you Akos, commit 4266b8e53034aadd3b49d03dbcbf55488e7ecd81